frontend/src/app/helpers/skeleton-loaders/mat-card-overview-loader/mat-card-overview-loader.component.ts

```diff
@@ -33,8 +35,14 @@ export class MatCardOverviewLoaderComponent implements OnInit {
   resize(width: number, height: number) {
     // Margin is 1vw (left + right) and 1vh (top + bottom)
 
-    const cardsPerColumn = (0.98 * width) / 425;
-    const cardsPerRow = (0.98 * height - 120) / 275;
+    let cardsPerColumn = 0;
+    if (this.rows) {
+      cardsPerColumn = this.rows;
+    } else {
+      cardsPerColumn = (0.98 * height - 120) / 275;
+    }
```

Comment:
```suggestion
    let cardsPerColumn = this.rows ? this.rows : (0.98 * height - 120) / 275;
```
That more or less a personal preference, that I really like conditional assignments in these cases